src/clj_3df/core.cljc

```diff
@@ -30,18 +30,18 @@
 (defprotocol IDB
   (-schema [db])
   (-attrs-by [db property])
-  (-attr->int [db attr])
-  (-int->attr [db i]))
+  (-attr->str [db attr])
```

Comment:
To make this patch useful in the long-term: Can we rename this to be `-attribute->id` and `-id->attribute`?

Reply:
Sure thing!
Makes sense